CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ We use [semantic versioning](http://semver.org/):
 - PATCH version when you make backwards compatible bug fixes.
 
 # Next Release
+- [breaking change] _teamscale-client_: Impacted tests now allows to specify multiple partitions
+- [breaking change] _teamscale-client_: Impacted tests baseline can now be a string
```

Comment:
What does this have to do with the customer request? And what's the benefit? Is that that you're now able to pass baseline names, like "Baseline Sprint 25"?

Reply:
I missed to add that to the ticket description, but that was also requested. It's mostly to allow specifying 123456p1 baselines, but also to allow HEAD for example